sdcm/utils/ldap.py

```diff
@@ -22,9 +22,9 @@
 LDAP_SSH_TUNNEL_SSL_PORT = 5002
 ORGANISATION = 'ScyllaDB'
 LDAP_DOMAIN = 'scylladb.com'
-LDAP_PASSWORD = 'scylla'
+LDAP_PASSWORD = 'cassandra'
 LDAP_ROLE = 'scylla_ldap'
-LDAP_USERS = ['scylla-qa', 'dummy-user']
+LDAP_USERS = ['cassandra', 'scylla-qa', 'dummy-user']
```

Comment:
Does it affect anyhow the ldap authorization?
@fgelcer ?

Reply:
from authorization perspective, it shouldn't make any difference, as there is already a "role" in Scylla, and this will be a user (from LDAP perspective) and not a role, so nothing to worry

Reply:
@fgelcer what I don't understand is if we have this user here and we also have the built in cassandra user, how will we know if it's actually use the LDAP user?

Reply:
well, i'm not sure you will be able to know where it is coming from, but the thing is, (IIUC) the "roles" in Scylla will be only roles for LDAP authentication, as the procedure of authenticating an user will come from LDAP only...
we could not have this cassandra in here at this moment, but it will all depend on how we manipulate the LDAP objects.

Reply:
I just checked and this will revert the change by @fgelcer that uses LDAP user in all places.

Reply:
remove it from here.

Reply:
> I just checked and this will revert the change by @fgelcer that uses LDAP user in all places.

ah well, for the authorization tests, this could simple have the order switched (to have `cassandra` user in any place of the list but 1st element)